### ISSUE TITLE
Help command presents wrong topics on subcommands

### ISSUE THREAD

**Issue by otaviojacobi** [balena-io/balena-cli#2741]
# Expected Behavior

`balena os configure --help` should display the usage as 

```
USAGE
  $ balena os configure <image>
```
# Actual Behavior

It currently displays: 
```
USAGE
  $ balena os:configure os configure <image>
```

**Comment by otaviojacobi** [comment]
Fixed on upstream @oclif/core v3.23.1

**Comment by otaviojacobi** [comment]
fixed with https://github.com/balena-io/balena-cli/pull/2739
